drivers/hmis/app/graphql/types/hmis_schema/enrollment.rb

```diff
@@ -168,6 +168,12 @@ def reminders
       Hmis::Reminders::ReminderGenerator.perform(project: project, enrollments: enrollments)
     end
 
+    def last_bed_night_date
+      return unless project.project_type == 1
```

Comment:
```suggestion
      return unless project.es_nbn?
```

Reply:
Also feeling like this is out of scope, and will add something in the 2024 branch